locust/test/test_main.py

```diff
@@ -191,6 +192,58 @@ def my_task(self):
             self.assertIn("Shutting down (exit code 0)", stderr)
             self.assertEqual(0, proc.returncode)
 
+    def test_percentile_parameter(self):
+        port = get_free_tcp_port()
+        with temporary_file(
+            content=textwrap.dedent(
+                """
+            from locust import User, task, constant, events
+            from locust.stats import PERCENTILES_TO_CHART
+            PERCENTILES_TO_CHART[0] = 0.9
+            PERCENTILES_TO_CHART[1] = 0.4
+            class TestUser(User):
+                wait_time = constant(3)
+                @task
+                def my_task(self):
+                    print("running my_task()")
+        """
+            )
+        ) as file_path:
+            proc = subprocess.Popen(
+                ["locust", "-f", file_path, "--web-port", str(port), "--autostart"], stdout=PIPE, stderr=PIPE, text=True
+            )
+            gevent.sleep(1)
+            proc.send_signal(signal.SIGTERM)
+            stdout, stderr = proc.communicate()
+            self.assertIn("Starting web interface at", stderr)
+            try:
+                response = requests.get(f"http://0.0.0.0:{port}/")
+                self.assertEqual(200, response.status_code)
```

Comment:
This doesnt really test anything? Any error would be caught..

Reply:
I'm expecting 200 response, I think I can remove try/exception

Reply:
I made mistake, I'll check this test again

Reply:
I wanted to check normal behavior in this test,  I tried requests.get(f"http://0.0.0.0:{port}/") and self.assertEqual(0, proc.returncode) to check, but none of them worked by running unittest. How can I check normal case? 
Anyway, test_webserver line 172 is actually testing normal case. Can I remove this test?

Reply:
proc.send_signal(signal.SIGTERM) will kill locust. Move that statement to the end of the test.

Reply:
That worked, I have pushed update. Thank you for helping a lot.